.readthedocs.yaml

```diff
@@ -15,7 +15,7 @@ build:
   jobs:
     post_install:
       # Build package with doc requirements from pyproject.optional-dependencies
-      - pip install --verbose --editable .[doc]
+      - pip install --verbose --editable .[docs]
```

Comment:
this would deviate PGM from PGM-IO. please either revert this or also create a PR there.

Reply:
Note: Using `docs` doesn't build, I just tried it out.

Reply:
Yup. Just tried again, `pip install --verbose --editable .[docs]` yields _WARNING: power-grid-model 1.9 does not provide the extra 'docs'_

Reply:
It should have worked, but reverted  it either way.